### PR TITLE
Closes #837: Adjust dataverse page to wcag rules

### LINKED ISSUE #837: Make dataverse page wcag complaint (to specific points listed in description)

### PR DESCRIPTION
**Pull request by madryk**

### CODE REVIEW

File: dataverse-persistence/src/main/resources/Bundle_pl.properties [hunk 1]
```diff
@@ -899,14 +914,26 @@ dataverse.search.facet.error=Wyst\u0105pi\u0142 b\u0142\u0105d dotycz\u0105cy Tw
...
+dataverse.results.cards.file.thumbnailPrefix=Miniaturka pliku
 dataverse.results.solrIsDown=Uwaga: Z powodu wewn\u0119trznego b\u0142\u0119du, przegl\u0105danie i wyszukiwanie nie jest dost\u0119pne.
+dataverse.results.page.previous=Poprzednia
+dataverse.results.page.next=Nast\u0119pna
+dataverse.results.page.screenReader.firstPrefix=pierwsza
+dataverse.results.page.screenReader.lastPrefix=ostatnia
+dataverse.results.page.screenReader=strona wynik\u00F3w wyszukiwania
+
+dataverse.results.page.previous=Poprzednia
+dataverse.results.page.previous.screenReader=
```
Comment (dbojanek) [inline dataverse-persistence/src/main/resources/Bundle_pl.properties]: Should this be empty?

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/query/SearchForTypes.java [hunk 2]
```diff
@@ -62,6 +62,18 @@ public SearchForTypes toggleType(SearchObjectType type) {
...
+        Set<SearchObjectType> newTypes = new HashSet<>(types);
+        newTypes.add(type);
+        return new SearchForTypes(newTypes);
+    }
+    
+    public SearchForTypes removeType(SearchObjectType type) {
+        Set<SearchObjectType> newTypes = new HashSet<>(types);
+        newTypes.remove(type);
+        return new SearchForTypes(newTypes);
+    }
```
Comment (dbojanek) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/query/SearchForTypes.java]: Can we have java docs for these? Or can you refactor names to indicate that new instance is being returned?

**Review by dbojanek — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/query/SearchForTypes.java (hunk 2 above)
Comment (madryk) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/search/query/SearchForTypes.java]: At the end I didn't used this functions - so I will remove them

File: dataverse-persistence/src/main/resources/Bundle_pl.properties (hunk 1 above)
Comment (madryk) [reply dataverse-persistence/src/main/resources/Bundle_pl.properties]: It is leftover. I will remove this

**Review by dbojanek — APPROVED** [review]

**Pull request merged**